etc/kayobe/kolla.yml

```diff
@@ -443,7 +443,8 @@ kolla_build_args:
 # List of names of additional host variables to pass through from kayobe hosts
 # to kolla-ansible hosts, if set. See also
 # kolla_overcloud_inventory_pass_through_host_vars_map.
-#kolla_overcloud_inventory_pass_through_host_vars_extra:
+kolla_overcloud_inventory_pass_through_host_vars_extra:
```

Comment:
I can't say I like this approach but I don't know of a better way to get kayobe to template some of the endpoints

Reply:
Could put it in kolla/globals.yml?

Reply:
Yup. Annoyingly I tried exactly that a while ago but it failed for an unrelated reason. Fixed now